Source/Core/AudioCommon/CMakeLists.txt

```diff
@@ -59,13 +59,14 @@ if(WIN32)
   target_link_libraries(audiocommon PRIVATE audiocommon_xaudio27)
 
   set(ENV{OPENALDIR} ${PROJECT_SOURCE_DIR}/Externals/OpenAL)
+  #Dynamically loads openal32.dll
```

Comment:
This should probably be made less ambiguous. This script itself doesn't dynamically load OpenAL at all. Also there should be a space after the `#`

Reply:
Would this be a better comment?:
`# The backend dynamically loads openal32.dll. This will pick up any system-installed library or a library in the same directory as Dolphin.`

Reply:
I would say "Dolphin loads openal32.dll at runtime" or something like that.

Reply:
@leoetlino Done